cmd/format.go

```diff
@@ -180,7 +180,7 @@ func fixObjectSize(s int) int {
 	return s
 }
 
-func CreateStorage(format meta.Format) (object.ObjectStorage, error) {
+func createStorage(format *meta.Format) (object.ObjectStorage, error) {
```

Comment:
we shouldn't pass pointer here, otherwise it will change the value of m.fmt, which will be used in DumpMeta.

Reply:
Only Decrypt() will change m.fmt, can DumpMeta work with decrypted Format?

Reply:
To be safe, I will revert the pointer here.